src/Components/NavBar/NotificationsBadge.tsx

```diff
@@ -92,35 +82,16 @@ const CircularCount: React.FC<{
    * Formatted count for display
    */
   count?: string
-  /**
-   * Raw unread count, used for analytics.
-   */
-  rawCount?: number
-  /**
-   * True if hovering over the badge
-   */
-  hover?: boolean
-}> = ({ count, rawCount, hover }) => {
+}> = ({ count }) => {
   // Check to see if we've got a value from sharify, populated by a cookie on
   // the server.
   const { notificationCount } = useContext(SystemContext)
   const notificationsLabel = count || notificationCount
-  const { trackEvent } = useTracking()
 
   if (!notificationsLabel) {
     return null
   }
 
-  useEffect(() => {
-    if (hover) {
-      trackEvent({
-        action_type: AnalyticsSchema.ActionType.Hover,
-        subject: AnalyticsSchema.Subject.NotificationBell,
-        new_notification_count: rawCount,
```

Comment:
Assuming it's ok to lose this flavor of event altogether, this LGTM 👍

Reply:
I tried the event with and without notifications. So I think so.

Reply:
I'm trying to think of a scenario where the notification count tracked by `NavBar` would be different than what the user actually saw, due to the cookie not being updated yet when they hovered. 

I *think* it's possible that the server could render a bell with 0 notifications, and while the [client request](https://github.com/artsy/reaction/pull/3050/files#diff-dbe27717b00f02b52447098fe8e92d13L22) was still waiting for a response from the server, the user hovered....in which case we would track that the user hovered over the bell with 0 notifications. But that would still be a correctly tracked event, since there were 0 listed *when they hovered*.

Are there any other scenarios either of you can dream up where communicating state between `NotificationsBadge` and `NavBar` via cookies would cause inaccurate events to be tracked?

Reply:
In general I think the way these components **should** be communicating state is with the parent (NavBar) managing it and passing it down. What we have right now, communicating state via cookies, seems to work...and it's less work than rewriting to manage state at the higher level. I'm good with this if we don't think we're opening ourselves up for weird bugs in terms of tracking the wrong thing.

Reply:
Ah, I hadn't even considered these subtleties. 

(Above I was responding to the [mistaken, I see now] impression that were were losing the `AnalyticsSchema.Subject.NotificationBell` type of hover event. But this is in fact still fired by the remaining code.)

@pepopowitz I think your take (i.e. that the edge case-y event would "still be a correctly tracked event") makes sense — I'm good with merging this.

Reply:
Okay in that case I'm going to go ahead and merge this.